test/integ/test_cursor.py

```diff
@@ -609,15 +609,119 @@ def test_geography(conn, db_parameters):
             cnx.cursor().execute(f"drop table {name_geo}")
 
 
+@pytest.mark.skipolddriver
 def test_callproc(conn_cnx):
-    """Callproc test.
+    """Callproc test."""
 
-    Notes:
-        It's a nop as of now.
-    """
+    def create_stored_procedure(cursor):
```

Comment:
This should use the [name randomizer](https://github.com/snowflakedb/snowflake-connector-python/blob/68d20293a342077e07d5981d1b197ec013e1f537/test/integ/test_put_get_user_stage.py#L206) and then you won't need to drop it first or replace it later.
This protects from the same tests running on different Python versions clashing

Reply:
updated